pyproject.toml

```diff
@@ -1,3 +1,7 @@
 [build-system]
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[tool.mypy]
+exclude = ["env/", ".env/","venv/", ".venv/", "build/"]
+follow_untyped_imports = true
```

Comment:
The [posix standard on lines](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) is rolling in its grave (I thought ruff accounted for this - maybe that's just for python files?).